lib/cogctl/actions/relay_groups/add.ex

```diff
@@ -2,27 +2,27 @@ defmodule Cogctl.Actions.RelayGroups.Add do
   use Cogctl.Action, "relay-groups add"
 
   def option_spec() do
-    [{:name, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
-     {:relay, :undefined, 'relay', {:string, :undefined}, 'Relay name (required)'}]
+    [{:relay_group, :undefined, :undefined, {:string, :undefined}, 'Relay Group name (required)'},
+     # Technically this will just be the first relay
```

Comment:
This part is a bit weird, and I'm not super happy with it, but I couldn't think of a better solution. At a minimum we require the name of the relay group and at least one relay, though there could be more, to add to the group.

Note: I handled bundle assignment the same way.